README.md

```diff
@@ -1 +1,202 @@
-# bookstore-react
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+Modify this file to match your project and remove sections that don't apply.
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Book Store <a name="about-project"></a>
+
+**Book store** is a website that you can use to storage books information.
+
+## 🛠 Built With <a name="built-with"></a>
+-webpack
+
+-npm
+
+-css
+
+-javascript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[]**
+
+- **[]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link][NOT AVALIABLE NOW]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ OPTIONAL] I noticed that there is a broken link in the README file `live demo link` , which may cause inconvenience to users who are trying to access the linked resource. :wink: 

I understand that mistakes can happen, but it is important to ensure that the README file contains accurate and up-to-date information. Using broken links not only creates a bad user experience but also affects the overall credibility of the project. please fix this :wink:  :+1: